src/wpgulp.config.js

```diff
@@ -22,6 +22,15 @@ module.exports = {
 	errLogToConsole: true,
 	precision: 10,
 
+	// Add-on Style options
+	// The following list is a set of SCSS/CSS files which you want to process and place it on a different folder.
+	addonStyles: [
+		{
+			styleSRC: './assets/css/addon.scss', // Path to .scss file.
+			styleDestination: './', // Path to place the compiled CSS file. Default set to root folder.
+		},
+	],
+
```

Comment:
I think by default there should be no addonStyles. Not everyone needs this. So, there should be an empty array with a link to the documentation. Which also means, in order to accept this PR, I need you to document at the end of the README.md file how to add more addon styles.